### PR TITLE
Search POC

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,10 +41,10 @@
     "react-emoji-render": "^2.0.1",
     "react-flagpack": "^2.0.5",
     "react-hook-form": "^7.53.0",
+    "react-hotkeys-hook": "^4.6.1",
```
Comment (haydencleary) [inline package.json:44]: Can use our current lib https://streamich.github.io/react-use/?path=/story/sensors-usekey--docs

File: app/pocs/search/components/search-bar/search-bar.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          <div
+            className={
+              "pointer-events-none absolute left-0 top-1/2 flex h-full w-full -translate-y-1/2 items-center px-2xl"
+            }
+          >
+            <p className={"text-[1rem] leading-[1.5rem] text-typography-secondary opacity-40"}>{prediction ?? ""}</p>
+          </div>
+        </div>
+      </div>
+      <FloatingPortal>
```
Comment (haydencleary) [inline app/pocs/search/components/search-bar/search-bar.tsx:83]: Could try this lib https://github.com/pacocoursey/cmdk

Comment (alexbeno) [reply app/pocs/search/components/search-bar/search-bar.tsx:83]: why it's for ?

File: package.json (hunk 1 above)
Comment (alexbeno) [reply package.json:44]: Not working for hotkeys it's only for for 1 key

File: app/pocs/search/components/search-bar/search-bar.tsx (hunk 2 above)
Comment (haydencleary) [reply app/pocs/search/components/search-bar/search-bar.tsx:83]: dialog/menu cmd k

Comment (alexbeno) [reply app/pocs/search/components/search-bar/search-bar.tsx:83]: We already use the eact-hotkeys-hook lib on marketplace

Comment (PierreOucif) [reply app/pocs/search/components/search-bar/search-bar.tsx:83]: Je valide !!!

Comment (haydencleary) [reply app/pocs/search/components/search-bar/search-bar.tsx:83]: lol

**Review by haydencleary — CHANGES_REQUESTED** [review]
will approve on monday

**Review by haydencleary — APPROVED** [review]

**Pull request merged**